addon/components/sortable-group.js

```diff
@@ -18,29 +19,35 @@ export default Component.extend({
     @property items
     @type Ember.NativeArray
   */
-  items: computed(() => { return a(); }),
+  items: computed({
+    get() { return a(); }
```

Comment:
Yeah that was the whole trigger to me creating PRs - I had deprecation warnings in my app about computed properties I wanted to get rid of.. 

Started with this one then thought 'why not add demoURL too' - since u already had the site as a demo.. :)